resources/app/src/components/people/DemographicsForm.vue

```diff
@@ -192,10 +192,21 @@
                     <br>
                     <h2>Participant Information on Age</h2>
                     <br>
-                    <div class="flex-container">
-                        <label>What year were you born?</label>
-                        <input class="w3-input" type="text" id="birth_year" v-model="formdata.birth_year"
+
+
+                    
+                    <div>
+                        <label for="specialty">Please select the year that you were born.</label>
+                        <span style="color: red !important; display: inline; float: none;"></span>
+                        <!-- TODO: check with invested parties: should this be multi-select/checkbox? -->
+                        <select id="age" name="age" v-model="formdata.birth_year"
```

Comment:
This variable should *not* be named age if it is the birth year! It would make no sense to be multi-select-- each person is only born in one year!

Reply:
I changed the variable to birth year.  I copied the template code, thus the reference to multi select.  Will remove.